plugins/woocommerce/client/blocks/package.json

```diff
@@ -242,7 +242,9 @@
 		"webpack-cli": "5.1.x",
 		"wireit": "0.14.12",
 		"wordpress-data-wp-6-7": "npm:@wordpress/data@wp-6.7",
-		"wp-types": "3.63.0"
+		"wp-types": "3.63.0",
+		"msw": "2.10.4",
+		"jest-fixed-jsdom": "0.0.9"
```

Comment:
Why do we need `jest-fixed-jsdom`? What Node globals are we using?

Reply:
I followed the msw documentation: https://mswjs.io/docs/faq/#requestresponsetextencoder-is-not-defined-jest

Reply:
It is necessary for `msw`. If I updated Jest configuration removing `jest-fixed-jsdom`, I got this error:
<img width="1285" height="291" alt="image" src="https://github.com/user-attachments/assets/efe99d08-b9b7-4d71-9fd9-17907d3c1382" />

Reply:
MSW docs also mention this: https://mswjs.io/docs/faq/#requestresponsetextencoder-is-not-defined-jest